rclcpp/include/rclcpp/service.hpp

```diff
@@ -141,6 +155,32 @@ class Service : public ServiceBase
     }
   }
 
+  Service(
+    std::shared_ptr<rcl_node_t> node_handle,
+    std::shared_ptr<rcl_service_t> service_handle,
+    AnyServiceCallback<ServiceT> any_callback)
+  : ServiceBase(node_handle),
+    any_callback_(any_callback)
+  {
+    // check if service handle was initialized
+    // TODO(karsten1987): Take this verification
+    // directly in rcl_*_t
+    // see: https://github.com/ros2/rcl/issues/81
```

Comment:
Looks like https://github.com/ros2/rcl/issues/81 is closed, can you try replacing this TODO and code snippet with `rcl_service_is_valid()`? http://docs.ros2.org/ardent/api/rcl/service_8h.html#ae3f8159e4c6c43f9f2cc10dd4c5f5f3f

Reply:
@Karsten1987 FYI

Reply:
done.